apps/expo/src/app/calendar/index.tsx

```diff
@@ -30,6 +30,13 @@ const Calendar = () => {
       day: new Date().toISOString().split("T")[0] ?? "", // today's date
     });
 
+  // Absences
+  const { data: absences } = api.absence.getAbsences.useQuery();
+
+  const absentActivityIds = new Set(
```

Comment:
So correct me if im wrong: a set is being used to store the IDs of the absence activities, however the absence schema contains information about the **current user** that is also absent. If one participant is absent for activity 1, and there are 2 participants, wouldn't activity 1 appear in this set? so then in your logic below when you render the absence, you do it if it is in this set, however, someone elses absence affects another person's display. 

Is my intuition wrong? please correct me if so!